open_api_framework/conf/base.py

```diff
@@ -950,25 +950,47 @@ def init_sentry(before_send: Callable | None = None):
 # ideally we'd use BASE_URI but it'd have to be lazy or cause issues
 CSP_DEFAULT_SRC = [
     "'self'",
-] + config("CSP_EXTRA_DEFAULT_SRC", default=[], split=True)
+] + config(
+    "CSP_EXTRA_DEFAULT_SRC",
+    default=[],
+    split=True,
+    help_text="Extra default source URLs for CSP other than ``self``. "
+    "Used for ``img-src``, ``style-src`` and ``script-src``",
+)
 
 CSP_REPORT_PERCENTAGE = config("CSP_REPORT_PERCENTAGE", 1.0)  # float between 0 and 1
 
 CSP_FORM_ACTION = (
     config(
         "CSP_FORM_ACTION",
         default=["\"'self'\""]
-        + config("CSP_EXTRA_FORM_ACTION", default=[], split=True),
+        + config(
+            "CSP_EXTRA_FORM_ACTION",
+            default=[],
+            split=True,
+            help_text="Add additional ``form-action`` source to the default ",
+        ),
         split=True,
+        help_text="Override the default ``form-action`` source",
```

Comment:
Are these necessary?

Reply:
yeah I would say so, either we should add `help_text` or set `add_to_docs=False` to avoid it from showing up in the envvar documentation. Let's also add `group="Content Security Policy"` to all of these `CSP_...` to make sure they show up in a separate section